src/utilities/generateChainExplorerUrl.ts

```diff
@@ -1,20 +1,30 @@
 import { CHAIN_METADATA } from 'constants/chainMetadata';
 import { ChainId } from 'types';
 
-export type UrlType = 'address' | 'token' | 'tx';
+export type UrlType = 'address' | 'token' | 'tx' | 'lztx';
```

Comment:
Just to be more explicit: 
```suggestion
export type UrlType = 'address' | 'token' | 'tx' | 'layerZeroTx';
```